spec/acceptance/windows_spec.rb

```diff
@@ -60,14 +56,17 @@
       inventory_hash = inventory_hash_from_inventory_file
       inventory_hash = remove_feature_from_group(inventory_hash, 'puppet-agent', 'winrm_nodes')
```

Comment:
we can replace with this
      inventory_hash = remove_feature_from_node(inventory_hash, 'puppet-agent', "#{ENV['TARGET_HOST']}")
      inventory_hash = remove_feature_from_node(inventory_hash, 'puppet-agent', 'local')